cluster-autoscaler/estimator/estimator_test.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package estimator
+
+import (
+	"testing"
+
+	kube_api "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/resource"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEstimate(t *testing.T) {
+
+	pod := &kube_api.Pod{
+		Spec: kube_api.PodSpec{
+			Containers: []kube_api.Container{
+				{
+					Resources: kube_api.ResourceRequirements{
+						Requests: kube_api.ResourceList{
+							kube_api.ResourceCPU:    *resource.NewMilliQuantity(500, resource.DecimalSI),
+							kube_api.ResourceMemory: *resource.NewQuantity(1000*1024*1024, resource.DecimalSI),
```

Comment:
How about we add a const for mem and cpu and use it here and below in assertions?